src/Microsoft.ApplicationInsights/Extensibility/TelemetryConfiguration.cs

```diff
@@ -269,7 +269,7 @@ public IApplicationIdProvider ApplicationIdProvider
         public EndpointContainer EndpointContainer { get; private set; } = new EndpointContainer(new EndpointProvider());
 
         /// <summary>
-        /// Gets or sets the connection string. Setting this value will also set the Instrumentation Key, validate the endpoints, and set the TelemetryChannel.Endpoint.
+        /// Gets or sets the connection string. Setting this value will also set (and overwrite) the Instrumentation Key. The endpoints are validated and will be set (and overwrite) our first party TelemetryChannels as well as the ApplicationIdProvider.
```

Comment:
will be set (and overwrite) our first party TelemetryChannels -> will be set (and overwritten) for InMemoryChannel and ServerTelemetryChannel